routes/helpers/load_balancer.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class Clover
+  def load_balancer_list
+    dataset = @project.load_balancers_dataset
+    dataset = dataset.authorized(current_account.id, "LoadBalancer:view")
+    dataset = dataset.join(:private_subnet, id: Sequel[:load_balancer][:private_subnet_id]).where(location: @location).select_all(:load_balancer) if @location
+    if api?
+      result = dataset.paginated_result(
+        start_after: request.params["start_after"],
+        page_size: request.params["page_size"],
+        order_column: request.params["order_column"]
+      )
+
+      {
+        items: Serializers::LoadBalancer.serialize(result[:records]),
+        count: result[:count]
+      }
+    else
+      @lbs = Serializers::LoadBalancer.serialize(dataset.all, {include_path: true})
+      view "networking/load_balancer/index"
+    end
+  end
+
+  def load_balancer_post(name: nil)
```

Comment:
All calls provide this argument.  While it doesn't need to done in this PR, we should probably switch this to a required positional argument.